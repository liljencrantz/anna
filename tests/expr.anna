
def main(){
    i:=0
    print(
	3,
	if(?){"a"} else {"b"},
	if(?){"X"},
	if(1){"a"},
	while(i<5){i++-i},
	while(i<5){'A'})
}
