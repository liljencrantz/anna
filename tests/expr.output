3 b ? a 0 ?
