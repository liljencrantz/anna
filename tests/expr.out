3bnulla-5null